madLibs.py

```diff
@@ -0,0 +1,24 @@
+"""
+author: kalez43@github.com
+description: This is a short Mad Libs game
+"""
+words = []
+
+
+# asks the user for inputs and puts them into a list
+def create_list():
+    inputs = ["Adjective: ", "Object: ", "Name: ", "Verb: ", "Number: ", "Body part: ", "Adverb: "]
+    for x in inputs:
+        word = input(x)  # asks for each input
+        word = word.upper()  # makes each input all caps
+        words.append(word)  # puts each input into a list
+
+
+create_list()  # runs the function
+

```

Comment:
You should pass the params to the function. Please update it

Reply:
what does that meannnnnnn

Reply:
It does not make sense to have this function have parameters, due to the nature of the game. There is only one way in which it would make sense, but as far as I can tell it is not possible to accomplish with what I know about python so far. Please call for further information